apps/dashboard/src/components/command-palette/command-types.ts

```diff
@@ -20,7 +20,6 @@ export interface Command {
   category: CommandCategory;
   keywords?: string[];
   icon?: ReactNode;
-  shortcut?: string;
```

Comment:
There is an actuall command under help section called "Keyboard Shortcuts" remove it. @cursoragent